crates/torii/core/src/engine.rs

```diff
@@ -305,16 +310,17 @@ impl<P: Provider + Sync> Engine<P> {
         }
 
         // Process blocks
-        for (block_number, block_timestamp) in blocks.iter() {
-            if let Some(ref block_tx) = self.block_tx {
-                block_tx.send(*block_number).await?;
-            }
+        // for (block_number, block_timestamp) in blocks.iter() {
+        //     if let Some(ref block_tx) = self.block_tx {
+        //         block_tx.send(*block_number).await?;
+        //     }
 
-            self.process_block(*block_number, *block_timestamp).await?;
-            info!(target: LOG_TARGET, block_number = %block_number, "Processed block.");
-        }
+        //     self.process_block(*block_number, *block_timestamp).await?;
+        //
+        // }
```

Comment:
this will be used later? If not, we can delete. May have a test specific for that if we're not sure about the removal.

Reply:
Ah yes we can remove.